core/lib/dependency-graph/network-node.js

```diff
@@ -25,14 +25,14 @@ class NetworkNode extends BaseNode {
    * @return {number}
    */
   get startTime() {
-    return this._record.startTime * 1000 * 1000;
+    return this._record.startTime * 1000;
```

Comment:
seems kinda awkward that the lantern depgraph and crc are still in seconds. 

wdyt about changing both of those in here, too?

Reply:
I mentioned this in the other PR, but I would like to keep lantern changes in a separate PR. I expect it to change a number of things in a trivial way (snapshots, accuracy) due to the limits of floating point precision. It should be a quick change I can follow up with when this lands.

Reply:
maybe add some comments in network-node/base-node/cpu-node and crc that these times are in seconds?
and/or a TODO?

cuz right now its kinda impossible to tell from the code.